packages/connect-common/files/firmware/t1b1/bitcoinonly/t1b1-1.11.2-bitcoinonly.json

```diff
@@ -0,0 +1,12 @@
+{
+    "required": false,
+    "version": [1, 11, 2],
+    "bootloader_version": [1, 11, 0],
+    "min_bridge_version": [2, 0, 25],
+    "min_firmware_version": [1, 6, 2],
+    "min_bootloader_version": [1, 5, 0],
+    "url": "/suite/firmware/t1b1/universal/t1b1-1.11.2-bitcoinonly.bin",
```

Comment:
soo, there is `universal` subfolder and `unsinged` subfolder?

Reply:
They are different things.

In testing environments we have `unsigned` and `signed` subfolders depending on the environment.

In production this PR is like the structure is going to look like: https://github.com/trezor/data/pull/140/files. So there will only be `universal` and `bitcoinonly` inside each device model. The reason to do that is to organize better the files, since having all the new files in one directory could be confusing. But we keep the location of all the already existing files for backward compatibility.